src/target/riscv/riscv.h

```diff
@@ -75,6 +75,7 @@ enum riscv_halt_reason {
 	RISCV_HALT_TRIGGER,
 	RISCV_HALT_UNKNOWN,
 	RISCV_HALT_GROUP,
+	RISCV_HALT_CRITERR,
```

Comment:
If you like - this will be more clear to code readers:

```suggestion
	RISCV_HALT_CRITICAL_ERROR,
```

Reply:
Addressed